.pylintrc

```diff
@@ -74,7 +74,8 @@ disable=missing-docstring,
         no-member,
         #TODO: Remove this once todos are done
         fixme,
-        consider-using-with
+        consider-using-with,
+        consider-using-f-string

```

Comment:
Pylint disable for now so we can unblock PRs without a failing master. We can add an issue to track if we merge